DataManagement/BranchTutorial.Rmd

```diff
@@ -84,165 +82,214 @@ knitr::include_graphics("Pics/gittutorial/14_git.png")
+Make a new **branch** and start all over again. 
 None of this will affect the original project.
 
+If you are working **on your own** project, and happy with the new code, or the idea is tested and working, you can **merge** the branch you are working on with the **main branch**.
+
+If you are **collaborating** with other people an you have forked a repo you do not want to work directly on the **main branch**. When you change the **main branch** and later update your repo from the original repo, it will cause **merge conflicts**. Therefore, when collaborating with others, you always work on **brnaches** and you can make a **pull requests** by **merging** the changes in the **current branch** to the **main branch** (see \@make-pull-request-in-github). 
+
 Let's have a look how to create, use and delete branches.
 
 
-### Make new branch (RStudio)
+### Make a new branch in RStudio
 
-In RStudio, click on the New Branch button in the Git window. 
-The box next to it will tell you in which branch you are. 
-Master is the default and original branch.
+In RStudio, click on the **New Branch** button in the Git tab. 
+Next to this button it will say on which branch you are.
+In this case it is main, which is the default branch.
 
 
 ```{r new-branch}
-knitr::include_graphics("Pics/7. NewBranch.png")
+knitr::include_graphics("Pics/fork_branch_tutorial/5_make_new_branch.png")
 ```
 <br/><br/>
 <br/><br/>
-Give the branch a short and informative name (no spaces) and click create.
+Give the branch a short and informative name (no spaces) and click **Create**.
 
 ```{r name-branch, out.width = '50%'}
-knitr::include_graphics("Pics/8. NameBranch.png")
+knitr::include_graphics("Pics/fork_branch_tutorial/6_name_branch.png")
 ```
 <br/><br/>
 <br/><br/>
-You can now see that you are in the new branch.
+You are now in the newly created branch.
 
 ```{r new-branch-2}
-knitr::include_graphics("Pics/9. NewBranch2.png")
+knitr::include_graphics("Pics/fork_branch_tutorial/7_new_branch.png")
 ```
 
 
-### Write R code, commit and push to origin (Rstudio)
+### Code, commit and push to origin in Rstudio
 
-Now is the time to open a new or an existing file and start to work on the project. 
-Work in the usual way, add small bits of code, commit regularly, add a useful commit message and push once in a while (see here for GitHub workflow).
-Once you are done with your project, test it and push it to your orgin repository on GitHub. 
-This works in the same way as when you work in your own repository.
+Now is the time work in a new or an existing file on the project. 
+You can work the way it is described in the \@how-to-work-in-git How to work with GitHub tutorial. Write or edit code, test it, **commit** regularly, add a useful commit message.
+
+Once you are done with the task in this branch, test it and **push** it to your **origin (remote?) repository** on GitHub. This is the workflow, if you are working on your own.
```

Comment:
remote?

Maybe call it the fork as well

Reply:
Does "forked repository" work?